src/components/Builder/Header/Header.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const Header = ({ rendered, image }) => {
```

Comment:
We have a `Header` component and there is a `<header>` in html which has different functionality than this, ie sets meta data... this should be something more along the lines of `Title` so maybe `ImageTilte` or `ClubImageTitle` depending if there is already a image title component or not.